vmaas/types.go

```diff
@@ -264,3 +267,17 @@ type NevraString struct {
 	Nevra utils.Nevra
 	Pkg   string
 }
+
+type CSAFProduct struct {
+	CpeID         CpeID
+	PackageNameID *int
+	PackageID     *int
```

Comment:
let's use types for package and package name ids
```suggestion
	PackageNameID *NameID
	PackageID     *PkgID
```